suncal/models/astro.py

```diff
@@ -101,33 +136,34 @@ def events(self):
         moonrise = moon_events['rise']
         moonset = moon_events['set']
 
-        return {
-            "sunrise": {
-                "start": sunrise,
-                "end": sunrise,
-                "gcal_summary": f"🌞↑ {sunrise.strftime('%I:%M %p')}"
-                if sunrise
-                else None,
-            },
-            "sunset": {
-                "start": sunset,
-                "end": sunset,
-                "gcal_summary": f"🌞↓ {sunset.strftime('%I:%M %p')}"
-                if sunset
-                else None,
-            },
-            "moonrise": {
+        if moonrise:
+            events['moonrise'] = {
                 "start": moonrise,
                 "end": moonrise,
-                "gcal_summary": f"🌜↑ {moonrise.strftime('%I:%M %p')}"
-                if moonrise
-                else None,
-            },
-            "moonset": {
+                "gcal_summary": f"🌜↑ {moonrise.strftime('%I:%M %p')}",
+            }
+        if moonset:
+            events["moonset"] = {
                 "start": moonset,
                 "end": moonset,
-                "gcal_summary": f"🌜↓ {moonset.strftime('%I:%M %p')}"
-                if moonset
-                else None,
-            },
-        }
+                "gcal_summary": f"🌜↓ {moonset.strftime('%I:%M %p')}",
+            }
```

Comment:
All four cases can be taken care in a loop over `zip([sunrise, sunset, moonrise, moonset], "sunrise sunset moonrise moonset".split())`

Reply:
True, thanks! I won't change this now, because I intend to refactor this part anyway.